src/Cello/mesh_Hierarchy.cpp

```diff
@@ -307,3 +307,81 @@ void Hierarchy::create_subblock_array
     
 }
 
+// --------------------------------------------------------------------
+
```

Comment:
Add comment to explain what function does

Reply:
Just had a look at this... it seems that this is the general pattern for the Hierarchy class, where descriptions of the member functions and attributes of the class are provided with the declarations in the header file, and there are no function descriptions provided in the ".cpp" file. I was probably just copying the pattern that I saw in the rest of the file.